gprofiler/profilers/python.py

```diff
@@ -279,6 +280,7 @@ def stop(self):
     "Python",
     possible_modes=["auto", "pyperf", "pyspy", "disabed"],
     default_mode="auto",
+    supported_archs=["x86_64", "aarch64"],  # we build pyspy for both, pyperf only for x86_64.
```

Comment:
mmm

Reply:
?

Reply:
Kinda wanky that we use this `ProfilerInterface` and have to lie to the registry instead of decorating the profiler classes directly with supported archs

Reply:
I didn't get it? Decorating the profiler with supported archs is actually good